Alignment/OfflineValidation/plugins/SplitVertexResolution.cc

```diff
@@ -295,12 +296,55 @@ SplitVertexResolution::SplitVertexResolution(const edm::ParameterSet& iConfig)
 
   mypT_bins_ = PVValHelper::makeLogBins<float, nPtBins_>(sumpTStartScale_, sumpTEndScale_);
 
-  std::vector<float> vect = PVValHelper::generateBins(nTrackBins_ + 1, 1., 120.);
+  // IMPORTANT
+  // first argument is start, second argument is the range so it's [-0.5;nTracksBins_-0.5]
```

Comment:
There are three arguments in fact: I guess the comment can be made more clear...